Templates/Ros2FleetRobotTemplate/Template/cmake/EngineFinder.cmake

```diff
@@ -0,0 +1,83 @@
+# This file is copied during engine registration. Edits to this file will be lost next
+# time a registration happens.
```

Comment:
Can you update the EngineFinder.cmake file here to use the latest version from the DefaultProject template: https://github.com/o3de/o3de/blob/development/AutomatedTesting/cmake/EngineFinder.cmake
Tagging @AMZN-alexpete

Reply:
I agree, please use the `EngineFinder.cmake` linked above which supports engine versioning and the ability for user's to set the path to their local engine in case they have multiple engine installs locally.

Reply:
Tested, with suggested file. Replaced.